activestorage/test/models/attached/many_test.rb

```diff
@@ -916,20 +916,28 @@ def self.name; superclass.name; end
     assert_match(/Cannot find variant :unknown for User#highlights_with_variants/, error.message)
   end
 
-  test "transforms variants later" do
-    blob = create_blob(filename: "funky.jpg")
+  test "transforms variants later: enqueues when blob is representable" do
+    representable_blob = create_file_blob(filename: "racecar.jpg")
 
-    assert_enqueued_with job: ActiveStorage::TransformJob, args: [blob, resize_to_limit: [1, 1]] do
-      @user.highlights_with_preprocessed.attach blob
+    assert_enqueued_with job: ActiveStorage::TransformJob, args: [representable_blob, resize_to_limit: [1, 1]] do
+      @user.highlights_with_preprocessed.attach representable_blob
+    end
+  end
```

Comment:
I don't think we need to make all these changes.  I think it is sufficient to only change `create_blob(filename: "funky.jpg")` to `create_file_blob(filename: "racecar.jpg")`, like you did for the other tests.